renderer/src/shared/TimeBadge/TimeBadge.tsx

```diff
@@ -20,8 +38,8 @@ export default function TimeBadge({ hours, startTime, selectedDate }: TimeBadgeP
     );
   }
   if (
-    (hours < 8 && selectedDate.toLocaleDateString() !== curDate.toLocaleDateString()) ||
-    (hours < 8 && curTime - stringToMinutes(startTime) > 480)
+    (hours + dayOffDuration < 8 && selectedDate.toLocaleDateString() !== curDate.toLocaleDateString()) ||
+    (hours + dayOffDuration < 8 && curTime - stringToMinutes(startTime) > 480)
```

Comment:
Could you, please, move 8 and 480 numbers to constants and call with a sense? 
Ex:
```
export const NAME_CONST = 8;
export const NAME_CONST  = 480;
```

Reply:
and 6 from 29-30 lines as well